internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -139,19 +139,19 @@ func (s *svc) templateConfiguration(tc templateConfigurer) (string, error) {
 	return doc.String(), nil
 }
 
-func (s *svc) addonsOutputs() (*template.ServiceNestedStackOpts, error) {
-	stack, err := s.addons.Template()
+func (w *wkld) addonsOutputs() (*template.ServiceNestedStackOpts, error) {
+	stack, err := w.addons.Template()
 	if err != nil {
 		var noAddonsErr *addon.ErrDirNotExist
 		if !errors.As(err, &noAddonsErr) {
-			return nil, fmt.Errorf("generate addons template for service %s: %w", s.name, err)
+			return nil, fmt.Errorf("generate addons template for service %s: %w", w.name, err)
```

Comment:
should we rename this error to be:
```suggestion
			return nil, fmt.Errorf("generate addons template for %s: %w", w.name, err)
```